assets/wishlist-page.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://component-library-priya.netlify.app/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-brand upper-text">
+            <a href="/home.html" class="nav-brand-link">Shoppers Stop</a></div>
+        <div class="nav-collapse">
+            <ul class="nav-list-group">
+                <li class="list-ele">
+                    <a href="/assets/signup.html" class="nav-link">Login</a>
+                </li>
+                <li class="list-ele">
+                    <a href="/assets/wishlist-page.html" class="nav-link">
+                        <i class="fa fa-heart-o"></i>
+                    </a>
+                </li>
+                <li class="list-ele">
+                    <a href="/assets/cart-page.html" class="nav-link">
+                        <i class="fa fa-shopping-cart"></i>Cart</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <main>
+        <section class="main-section">
+            <div class="prod-heading upper-text">My Wishlist</div>
+            <div class="wishlist-container">
+                <div class="card">
+                    <div class="container">
+                        <div class="image">
+                            <img src="/assets/home/image-4.jpg" alt="sunset-and-glass-mug" class="img">
+                            <div class="img-badge">
+                                <i class="fa fa-heart"></i>
+                            </div>
```

Comment:
You can make the badge circular by applying the bellow styles.
` .img-badge{`
    `padding: 0.5rem;`
    `top: 0.2rem;`
    `right: 0.2rem;`
    `align-items: center;`
    `justify-content: center;`
   `display: flex;`
    `width: 1rem;`
    `height: 1rem;`
`}`

Reply:
Great, I will definitely try it